### ISSUE TITLE
Faulty core fixes: space isn't added after a comma in a function call

### ISSUE THREAD

**Issue by pento** [WordPress-Coding-Standards/WordPress-Coding-Standards#1189]
Given the following test case:

```php
<?php

a( 1,2 );
```

The lack of space after the comma in the function call isn't detected.

Core ref: src/wp-admin/install-helper.php

Tested with PHPCS `master`, WPCS `develop`.

**Comment by jrfnl** [comment]
This should be sorted via PR #1193 ;-)
